deployments/aws/templates/ai-unlimited/ai-unlimited-with-nlb.yaml

```diff
@@ -684,7 +789,19 @@ Resources:
   AiUnlimitedRolePolicies:
     Type: AWS::IAM::Policy
     Properties:
-      PolicyName: !Join ['-', [ai-unlimited, deploy, !Select [4, !Split ['-', !Select [2, !Split [/, !Ref 'AWS::StackId']]]]]]
+      PolicyName: !Join
+        - '-'
+        - - ai-unlimited
+          - deploy
+          - !Select
+            - 4
+            - !Split
+              - '-'
+              - !Select
+                - 2
+                - !Split
+                  - /
+                  - !Ref AWS::StackId
       PolicyDocument:
```

Comment:
Please ensure that ec2:DescribeVpcAttribute and ec2:DescribeAvailabilityZones are included on PolicyDocument Statements.